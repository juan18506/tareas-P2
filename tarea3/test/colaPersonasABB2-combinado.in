crearCola

imprimirCantidadCola

crearArbolPersonas

# Encolo el arbol vacio
encolar

crearPersona 5 11 Maria
insertarPersonaArbolPersonas

crearPersona 4 55 Raul
insertarPersonaArbolPersonas

# Encola el arbol 
encolar

encolar
encolar
encolar

imprimirCantidadCola

# Obtengo el frente de la cola
obtenerFrenteCola

# Elimino de la Cola el Arbol con Persona 4 y 5
desencolar

imprimirCantidadCola

# Obtengo el frente de la cola
obtenerFrenteCola

# Elimino de la Cola el Arbol vacio
desencolar

imprimirCantidadCola

# Obtengo el frente de la cola
obtenerFrenteCola

# Elimino de la Cola el Arbol con Persona 1
desencolar

imprimirCantidadCola

liberarCola

liberarArbolPersonas

crearCola

# Creo arbol con Persona 2
crearArbolPersonas

crearPersona 2 56 Pablo
insertarPersonaArbolPersonas

# Encolo el arbol con Persona 2
encolar

imprimirCantidadCola

# Obtengo el frente de la Cola
obtenerFrenteCola

# Elimino de la Cola el Arbol con Persona 2
desencolar

liberarCola

liberarArbolPersonas

crearCola

# Creo arbol vacio
crearArbolPersonas

# Encolo el arbol vacio
encolar

imprimirCantidadCola

encolar
encolar
encolar
encolar
encolar
encolar
encolar
encolar
encolar
encolar
encolar
encolar

imprimirCantidadCola

liberarCola

liberarArbolPersonas

Fin