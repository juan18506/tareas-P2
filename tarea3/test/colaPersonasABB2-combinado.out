1>2>La cantidad de arboles de personas en la cola es: 0.
3>4>#  Encolo el arbol vacio.
5>6>7>8>9>10>#  Encola el arbol .
11>12>13>14>15>La cantidad de arboles de personas en la cola es: 5.
16>#  Obtengo el frente de la cola.
17>Arbol vacio
18>#  Elimino de la Cola el Arbol con Persona 4 y 5.
19>20>La cantidad de arboles de personas en la cola es: 4.
21>#  Obtengo el frente de la cola.
22>Persona 4:  Raul, 55 años
Persona 5:  Maria, 11 años
23>#  Elimino de la Cola el Arbol vacio.
24>25>La cantidad de arboles de personas en la cola es: 3.
26>#  Obtengo el frente de la cola.
27>Persona 4:  Raul, 55 años
Persona 5:  Maria, 11 años
28>#  Elimino de la Cola el Arbol con Persona 1.
29>30>La cantidad de arboles de personas en la cola es: 2.
31>32>33>34>#  Creo arbol con Persona 2.
35>36>37>38>#  Encolo el arbol con Persona 2.
39>40>La cantidad de arboles de personas en la cola es: 1.
41>#  Obtengo el frente de la Cola.
42>Persona 2:  Pablo, 56 años
43>#  Elimino de la Cola el Arbol con Persona 2.
44>45>46>47>48>#  Creo arbol vacio.
49>50>#  Encolo el arbol vacio.
51>52>La cantidad de arboles de personas en la cola es: 1.
53>54>55>56>57>58>59>60>61>62>63>64>65>La cantidad de arboles de personas en la cola es: 13.
66>67>68>Fin.
